tests/settings.py

```diff
@@ -59,6 +59,8 @@
     "resolwe.permissions.permissions.ResolwePermissionBackend",
 )
 
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
This is already the default value: https://docs.djangoproject.com/en/4.0/releases/3.2/#customizing-type-of-auto-created-primary-keys

Reply:
Either of them is fine, but I would go with the second one to follow Django.

Reply:
Sure, that is better in the long run.